api/src/pcapi/core/mails/transactional/bookings/booking_postponed_by_pro_to_beneficiary.py

```diff
@@ -0,0 +1,59 @@
+from typing import Union
+
+from babel.dates import format_date
+
+from pcapi.core import mails
+from pcapi.core.bookings.models import Booking
+from pcapi.core.mails.transactional.sendinblue_template_ids import SendinblueTransactionalEmailData
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.models.feature import FeatureToggle
+from pcapi.utils.mailing import format_booking_hours_for_email
+from pcapi.utils.mailing import get_event_datetime
+
+
+def send_batch_booking_postponement_email_to_users(bookings: list[Booking]) -> list[bool]:
+    for booking in bookings:
+        send_booking_postponement_email_to_users(booking)
```

Comment:
Cette fonction devrait renvoyer un booléen indiquant si tous les mails ont pu être envoyés. Ainsi l'appelant peut logger les erreurs ou avoir un comportement différent en fonction du retour.

```suggestion
def send_batch_booking_postponement_email_to_users(bookings: list[Booking]) -> bool:
    success = True
    for booking in bookings:
        success &= send_booking_postponement_email_to_users(booking)
    return success
```